File: python/ifelifelse/desafio5.py
     # 5) Faça um programa que leia uma variável e some 5 caso seja par ou some 8 caso
     #    seja ímpar. Imprimir o resultado desta operação.

from turtle import st


a = input("Digite um valor: ")

if(int(a) % 2 == 0):
    soma = int(a)+5
    print("O número "+str(a)+" é par e")
    print("A soma do número: "+str(a)+" + 5 é: "+str(soma))
else:
    soma = int(a)+8
    print("O número "+str(a)+" é ímpar e")
    print("A soma do número: "+str(a)+" + 8 é: "+str(soma))

File: java/lacoscondicionais/desafio5.java
package java.lacoscondicionais;

    /*
     *@autor: Hylan Silva
     *@version: 0.0.6 
     */

     import java.util.Scanner;

public class desafio5 {

     // 5) Faça um programa que leia uma variável e some 5 caso seja par ou some 8 caso
     //    seja ímpar. Imprimir o resultado desta operação.

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int a;

        System.out.print("Digite um número: ");
        a = in.nextInt();

        if(a % 2 == 0){
            System.out.println("O número "+a+" é par e");
            System.out.println("A soma do número: "+a+" + 5 é: "+a+8);
        }else{
            System.out.println("O número "+a+" é ímpar e");
			System.out.println("A soma do número: "+a+" + 8 é: "+(a+8));
        }
        in.close();
    }
}
